src/devices/Ssd1351/Ssd1351Commands.cs

```diff
@@ -0,0 +1,541 @@
+        /// </summary>
+        /// <param name="displayClockDivideRatio">Display clock divide ratio with a range of 0-15.</param>
+        /// <param name="oscillatorFrequency">Oscillator frequency with a range of 0-15.</param>
+        public void SetDisplayClockDivideRatioOscillatorFrequency(byte displayClockDivideRatio = 0x01, byte oscillatorFrequency = 0x0D)
+        {
+            if (displayClockDivideRatio > 0x0F)
+            {
+                throw new ArgumentException("The display clock divide ratio is invalid.", nameof(displayClockDivideRatio));
+            }
+
+            if (oscillatorFrequency > 0x0F)
+            {
+                throw new ArgumentException("The oscillator frequency is invalid.", nameof(oscillatorFrequency));
+            }
+
+            SendCommand(Ssd1351Command.SetClockDiv, (byte)((oscillatorFrequency << 4) | displayClockDivideRatio));
+        }
+
+        /// <summary>
+        /// This command is used to set Contrast Setting of the display.
+        /// The chip has 256 contrast steps from 00h to FFh.  The segment
+        /// output current ISEG increases linearly with the contrast step,
+        /// which results in brighter display.
+        /// </summary>
+        /// <param name="colorAContrast">Contrast level for color A.</param>
+        /// <param name="colorBContrast">Contrast level for color B.</param>
+        /// <param name="colorCContrast">Contrast level for color C.</param>
+        public void SetContrastABC(byte colorAContrast = 0x86, byte colorBContrast = 0x51, byte colorCContrast = 0x86)
+        {
+            SendCommand(Ssd1351Command.SetContrastABC, colorAContrast, colorBContrast, colorCContrast);
+        }
+
+        /// <summary>
+        /// This command specifies the mapping of the display start line to one of COM0-COM127
+        /// (assuming that COM0 is the display start line then the display start line register is equal to 0).
+        /// </summary>
+        /// <param name="displayOffset">Display offset with a range of 0-127.</param>
+        public void SetDisplayOffset(byte displayOffset = 0x60)
+        {
+            if (displayOffset > 0x7F)
+            {
+                throw new ArgumentException("The display offset is invalid.", nameof(displayOffset));
+            }
+
+            SendCommand(Ssd1351Command.SetDisplayOffset, displayOffset);
+        }
+
+        /// <summary>
+        /// This command sets the Display Start Line register to determine starting address of display RAM,
+        /// by selecting a value from 0 to 127. With value equal to 0, RAM row 0 is mapped to COM0.
+        /// With value equal to 1, RAM row 1 is mapped to COM0 and so on.
+        /// </summary>
+        /// <param name="displayStartLine">Display start line with a range of 0-127.</param>
+        public void SetDisplayStartLine(byte displayStartLine = 0x00)
+        {
+            if (displayStartLine > 0x7F)
```

Comment:
I won't comment on the lower band check anymore but assume we have to fix this on all subsequent cases.